core/kazoo_data/src/kzs_plan.erl

```diff
@@ -292,6 +336,36 @@ fetch_cached_dataplan(Key, _Fun) ->
         {'error', 'not_found'} -> ?CACHED_SYSTEM_DATAPLAN
     end.
 
+-spec fetch_cached_dataconnection(kz_term:ne_binary(), map()) -> server() | {'error', 'not_found'}.
```

Comment:
Spec is wrong. I see callers with tuples for second arg (see `system_dataplan/0` for instance).